moa/src/main/java/moa/streams/filters/HashingTrickFilter.java

```diff
@@ -0,0 +1,110 @@
+
+
+package moa.streams.filters;
+import com.github.javacliparser.IntOption;
+import com.google.common.hash.Hashing;
```

Comment:
You forget to include the guava dependency.
At `moa/moa/pom.xml` add the following dependency:
```xml
    <dependency>
      <groupId>com.google.guava</groupId>
      <artifactId>guava</artifactId>
      <version>29.0-jre</version>
    </dependency>
```